src/Bem.js

```diff
@@ -3,6 +3,28 @@ import stringify from '@bem/sdk.naming.entity.stringify';
 
 export default function({ preset, naming }) {
     const { Base, classAttribute, Render, PropTypes } = preset;
+    const getRenderProps = function(instance, node) {
+        const bemModes = {
```

Comment:
Dont need to create this hash each call, move it to top
Also we don't need `prototype` for this hash so we can create object via `Object.create(null)`
Or even we can just use Set: `new Set` and then check property for being a bemMode via .has:
`if(!bemModes.has(p)) props[p] = node[p]`

Reply:
Agree with moving to the top, but what about `Set`? Does it supported well in browsers. Because it will be big awful construction if we use babel for it.